Static/DataSource.swift

```diff
@@ -188,11 +187,11 @@ extension DataSource: UITableViewDataSource {
     }
 
     public func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
     }
     
     public func tableView(tableView: UITableView, estimatedHeightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
```

Comment:
The API exists so that the scroll indicator is sized correctly. If the consumer of `Static` is using autolayout, `UITableViewAutomaticDimension` then setting an explicit estimated height will determine the size of the scroll indicator. 

On the other hand, if the consumer does not set an explicit estimated height, and chooses to use `UITableViewAutomaticDimension` instead, all of the heights for all of the cells need to be calculated via autolayout by the system in order to determine the size of the scroll indicator. This could be a while.
By always setting `estimatedHeight` to `UITableViewAutomaticDimension`, we are forcing the latter, less performant, situation whenever the consumer chooses to use autolayout.